src/Appwrite/Utopia/Response/Model/Execution.php

```diff
@@ -16,6 +16,13 @@ public function __construct()
                 'default' => '',
                 'example' => '5e5ea5c16897e',
             ])
+            ->addRule('$permissions', [
```

Comment:
Why is the reason behind this addition?

Reply:
I need to have some type of `permissions` value seeing who is to receive an event for Executions 🙂